controllers/errorHandlerControllers.js

```diff
@@ -9,6 +13,8 @@ exports.sqlErrorHandler = (error, request, response, next) => {
 exports.customErrorHandler = (error, request, response, next) => {
 	if (error === "no content") {
 		response.status(204).send();
+	} else if (error === "article not found") {
+		response.status(404).send({ msg: error });
 	} else {
 		console.log(error);
```

Comment:
I might be tempted to extract the 500s out into their own error handler as its kind of not dealing with our custom errors, and will cause problems if you try adding anything in the chain after this.